### ISSUE TITLE
Add 'select all' for tags when adding a resource

### ISSUE THREAD

**Issue by andoneve** [mokus80/learners_directory#16]
When you're posting a resource, include a 'select all' option for tags.

**Comment by PragTob** [comment]
Also the question if you mean all possible tags. If so, I don't think it's a good option as I understand it. E.g. to my mind a lot of the tags are mutually exclusive (Rails 32. vs. 4.0, Beginner vs. intermediate vs. expert)

**Comment by andoneve** [comment]
They aren't mutually exclusive because a lot of resources have levels for
all different types of learners - and the Ruby on Rails tutorial for
instances has a version for Rails 3.2 as well as Rails 4.0.

On Thu, Nov 7, 2013 at 1:42 PM, Tobias Pfeiffer notifications@github.comwrote:

> Also the question if you mean all possible tags. If so, I don't think it's
> a good option as I understand it. E.g. to my mind a lot of the tags are
> mutually exclusive (Rails 32. vs. 4.0, Beginner vs. intermediate vs. expert)
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/mokus80/learners_directory/issues/16#issuecomment-27959303
> .

## 

laura pierson wadden
+49 0176 8773 3595

**Comment by PragTob** [comment]
Yes there are such resources, in my experience that's just not the majority of resources. That the Railstutorial is for multiple versions is extraordinary. Every book I ever read about Rails just considers one specific version. Same is true skill wise for most resources - e.g. I know no resources that are both beginner/introduction level and expert level. I know some resource collection that have beginner and expert material. Also topic wise a lot of resources focus on a couple of topics (Again, the excellent rails guide is an exception).

Just saying that personally I doubt that selecting all tags is one of the more common use case. And I don't want us to get feature creep :-)

**Comment by andoneve** [comment]
yes, that makes sense! thanks. Just was capturing some feedback from a
friend who tried to use the site.

On Sun, Nov 10, 2013 at 9:06 PM, Tobias Pfeiffer
notifications@github.comwrote:

> Yes there are such resources, in my experience that's just not the
> majority of resources. That the Railstutorial is for multiple versions is
> extraordinary. Every book I ever read about Rails just considers one
> specific version. Same is true skill wise for most resources - e.g. I know
> no resources that are both beginner/introduction level and expert level. I
> know some resource collection that have beginner and expert material. Also
> topic wise a lot of resources focus on a couple of topics (Again, the
> excellent rails guide is an exception).
> 
> Just saying that personally I doubt that selecting all tags is one of the
> more common use case. And I don't want us to get feature creep :-)
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/mokus80/learners_directory/issues/16#issuecomment-28159159
> .

## 

laura pierson wadden
+49 0176 8773 3595

**Comment by PragTob** [comment]
User testing - jay! =)
